modules/update-common-terms.adoc

```diff
@@ -19,8 +19,8 @@ Channels:: _Channels_ declare an update strategy tied to minor versions of {prod
 Recommended update edge:: A _recommended update edge_ is a recommended update between {product-title} releases.  Whether a given update is recommended can depend on the cluster's configured channel, current version, known bugs, and other information. OSUS communicates the recommended edges to the CVO, which runs in every cluster.
 
 ifndef::openshift-origin[]
-
-Extended Update Support:: All post-4.7 even-numbered minor releases are labeled as _Extended Update Support_ (EUS) releases. These releases introduce a verified update path between EUS releases, permitting customers to streamline updates of worker nodes and formulate update strategies of EUS-to-EUS {product-title} releases that result in fewer reboots of worker nodes.
+// do we need to change this at all?
```

Comment:
I agree, we can get rid of this. But we should get rid of it from line 21 to 27 as the whole ifndef/endif block is together.